SpywareFilter/sections/mobile_allowlist.txt

```diff
@@ -16,8 +16,6 @@
 @@||tr.line.me^$app=jp.co.alc.eow
 @@||tr.outbrain.com^$app=jp.co.alc.eow
 ! https://github.com/AdguardTeam/AdguardFilters/issues/81513
-@@||google-analytics.com/analytics.js$script,domain=m.dcinside.com|mupload.dcinside.com
-@@||ssl.google-analytics.com/ga.js$script,domain=m.dcinside.com|mupload.dcinside.com
```

Comment:
Better wrap by the directive. Exclusions still required for `content blocker for Android`(not the AdGaurd for Android app).
```
!#if (adguard_ext_android_cb)
@@||google-analytics.com/analytics.js$script,domain=m.dcinside.com|mupload.dcinside.com
@@||ssl.google-analytics.com/ga.js$script,domain=m.dcinside.com|mupload.dcinside.com
!#endif
```